backend/src/LibCloud/SqlCompiler.fs

```diff
@@ -739,6 +686,21 @@ let rec lambdaToSql
           | _ ->
             return
               error $"We do not support this type of DB field yet: {dbFieldType}"
+        | EIf(_, cond, then_, else_) ->
```

Comment:
I needed support for `EIf` here to be able to inline `Package.*.Bool.not/and/or` and `Package.*.String.contains`. Is there a reason why we didn't support it before?

Reply:
`Bool.and` should be implemented as `a && b` and then I think the inlining will be simpler. Same for `Bool.or`: `a || b`.